openapi/server.yml

```diff
@@ -0,0 +1,285 @@
+openapi: 3.1.0
+info:
+  title: Y-Sweet Server API
+  version: '0.5.0'
+  x-logo:
+    url: 'https://raw.githubusercontent.com/jamsocket/y-sweet/main/logo.svg'
+  description: |
+    Y-Sweet has two APIs:
+    1. **A document management API** (this one), which allows creating and updating documents.
+    2. **A single document API**, which allows connecting to and updating a single document.
+
+    This approach allows for the document management server to live outside of the “data plane” in a multi-server setup,
+    so that none of the actual document data (i.e. the Yjs document updates) ever passes through the document management API.
+
+    The API described here is the document management API. A caller can use this API to obtain a `baseUrl` for a given document,
+    which can then be used as a base for the [single document API](/document.html).
+paths:
+  /doc/new:
+    post:
+      summary: Create New Document
+      description: Creates a new document. Optionally accepts a `docId`.
+      security:
+        - bearerAuth: []
+      requestBody:
+        required: false
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                docId:
+                  type: string
+      responses:
+        '200':
+          description: Document created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  docId:
+                    type: string
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+  /doc/{docId}/auth:
+    post:
+      summary: Generate Client Token
+      description: Generates a client token for accessing a document.
+      security:
+        - bearerAuth: []
+      parameters:
+        - in: path
+          name: docId
```

Comment:
Doesn't this endpoint also take a body with `authorization`, `user_id`, and `metadata` fields?